### PR TITLE
Resize

### PR DESCRIPTION
**Pull request by TomBrouws**
Allow the user to resize the application, up to a minimum of 800x600 resolution.

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/Application.java [hunk 1]
```diff
@@ -21,7 +21,9 @@
 import tudelft.ti2806.pl3.zoomBar.ZoomBarController;
 import tudelft.ti2806.pl3.zoomBar.ZoomBarView;
 
-import java.awt.Component;
+import java.awt.*;
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/Application.java]: Unused imports within * import.

File: src/main/java/tudelft/ti2806/pl3/Application.java [hunk 2]
```diff
@@ -87,6 +90,8 @@ public void make() {
 		addWindowListener(windowController);
 		loadingObservers.add(new LoadingMouse(this));
 
+		this.setMinimumSize(new Dimension(800, 600));
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/Application.java]: Add some constants for this?

File: src/main/java/tudelft/ti2806/pl3/sidebar/phylotree/PhyloView.java [hunk 3]
```diff
@@ -92,6 +95,14 @@ private void setUI(int width, int height) {
 		setPreferredSize(new Dimension(width, height));
 	}
 
+	public void updateSize() {
+		int width = ScreenSize.getInstance().getSidebarWidth() - 10;
+		int height = ScreenSize.getInstance().getHeight() - 100;
+
+		scroller.setPreferredSize(new Dimension(width, (int) (height / 1.1)));
+		header.setPreferredSize(new Dimension(width, 50));
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/sidebar/phylotree/PhyloView.java]: Magic numbers

File: src/main/java/tudelft/ti2806/pl3/sidebar/phylotree/PhyloView.java [hunk 4]
```diff
@@ -92,6 +95,14 @@ private void setUI(int width, int height) {
 		setPreferredSize(new Dimension(width, height));
 	}
 
+	public void updateSize() {
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/sidebar/phylotree/PhyloView.java]: javadoc missing

**Comment by Delfil** [comment]
Everything else is ok.

**Comment by mathieupost** [comment]
Nice! Merge!

**Pull request merged**